torch/custom_class_detail.h

```diff
@@ -135,7 +135,6 @@ inline void checkValidIdent(const std::string& str, const char *type) {
 }
 
 } // namespace detail
-
```

Comment:
maybe revert this change?

Reply:
Good catch. Just updated it, and also rebase to internal fbobjc/stable for easier landing.